tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -500,6 +500,7 @@ describe("Service Form Modal", function() {
             .focus()
             .type("{selectall}{backspace}")
             .type("{selectall}{backspace}")
+            .type("{selectall}{backspace}")
```

Comment:
Locally, this prevents Cypress from typing part of the input at the beginning of the field. This is more of a workaround than an actual fix, but I want to see if it works in CI